Patch-Source: https://github.com/php/php-src/pull/8617
From 1bb6d51629b0dba5a3d21a8c95329c2141097db7 Mon Sep 17 00:00:00 2001
From: Andy Postnikov <apostnikov@gmail.com>
Date: Tue, 24 May 2022 02:12:58 +0300
Subject: [PATCH] Make socket path shorter for
 ext/sockets/tests/socket_cmsg_{rights|credentials}.phpt

When running in CI it fails when path/address is longer 108
---
 ext/sockets/tests/socket_cmsg_credentials.phpt | 4 ++--
 ext/sockets/tests/socket_cmsg_rights.phpt      | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/ext/sockets/tests/socket_cmsg_credentials.phpt b/ext/sockets/tests/socket_cmsg_credentials.phpt
index 7402c9437c2b..5a9efc269e77 100644
--- a/ext/sockets/tests/socket_cmsg_credentials.phpt
+++ b/ext/sockets/tests/socket_cmsg_credentials.phpt
@@ -17,7 +17,7 @@ die('skip SO_PASSCRED is not defined');
 --FILE--
 <?php
 include __DIR__."/mcast_helpers.php.inc";
-$path = __DIR__ . "/socket_cmsg_credentials.sock";
+$path = "./socket_cmsg_credentials.sock";
 
 @unlink($path);
 
@@ -55,7 +55,7 @@ var_dump($data['control'][0]['data']['pid'] === $pid);
 ?>
 --CLEAN--
 <?php
-$path = __DIR__ . "/socket_cmsg_credentials.sock";
+$path = "./socket_cmsg_credentials.sock";
 @unlink($path);
 --EXPECTF--
 creating send socket
diff --git a/ext/sockets/tests/socket_cmsg_rights.phpt b/ext/sockets/tests/socket_cmsg_rights.phpt
index 2c45e449bdc3..264f59967057 100644
--- a/ext/sockets/tests/socket_cmsg_rights.phpt
+++ b/ext/sockets/tests/socket_cmsg_rights.phpt
@@ -13,12 +13,12 @@ die('skip not for AIX');
 }
 --CLEAN--
 <?php
-$path = __DIR__ . "/socket_cmsg_rights.sock";
+$path = "./socket_cmsg_rights.sock";
 @unlink($path);
 --FILE--
 <?php
 include __DIR__."/mcast_helpers.php.inc";
-$path = __DIR__ . "/socket_cmsg_rights.sock";
+$path = "./socket_cmsg_rights.sock";
 
 @unlink($path);
 
